website/content/docs/k8s/operations/uninstall.mdx

```diff
@@ -6,73 +6,109 @@ description: Uninstall Consul on Kubernetes
-  To delete, run:
-
-  ```shell-session
-  $ kubectl get pvc -l chart=consul-helm
-  NAME                                   STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
-  data-default-hashicorp-consul-server-0   Bound    pvc-32cb296b-1213-11ea-b6f0-42010a8001db   10Gi       RWO            standard       17m
-  data-default-hashicorp-consul-server-1   Bound    pvc-32d79919-1213-11ea-b6f0-42010a8001db   10Gi       RWO            standard       17m
-  data-default-hashicorp-consul-server-2   Bound    pvc-331581ea-1213-11ea-b6f0-42010a8001db   10Gi       RWO            standard       17m
-
-  $ kubectl delete pvc -l chart=consul-helm
-  persistentvolumeclaim "data-default-hashicorp-consul-server-0" deleted
-  persistentvolumeclaim "data-default-hashicorp-consul-server-1" deleted
-  persistentvolumeclaim "data-default-hashicorp-consul-server-2" deleted
-  ```
-
-  ~> **NOTE:** This will delete **all** data stored in Consul and it can't be
-     recovered unless you've taken other backups.
-
-  3. If installing with ACLs enabled, you will need to then delete the ACL secrets:
-
-  ```shell-session
-  $ kubectl get secret | grep consul | grep Opaque
-  consul-acl-replication-acl-token    Opaque                                1      41m
-  consul-bootstrap-acl-token          Opaque                                1      41m
-  consul-client-acl-token             Opaque                                1      41m
-  consul-connect-inject-acl-token     Opaque                                1      37m
-  consul-controller-acl-token         Opaque                                1      37m
-  consul-federation                   Opaque                                4      41m
-  consul-mesh-gateway-acl-token       Opaque                                1      41m
-  ```
-
-  4. Ensure that the secrets you're about to delete are all created by Consul and not 
-  created by someone else that happen to have the word `consul`:
-
-  ```shell-session
-  $ kubectl get secret | grep consul | grep Opaque | awk '{print $1}' | xargs kubectl delete secret
-  secret "consul-acl-replication-acl-token" deleted
-  secret "consul-bootstrap-acl-token" deleted
-  secret "consul-client-acl-token" deleted
-  secret "consul-connect-inject-acl-token" deleted
-  secret "consul-controller-acl-token" deleted
-  secret "consul-federation" deleted
-  secret "consul-mesh-gateway-acl-token" deleted
-  secret "consul-gossip-encryption-key" deleted
-  ```
-
-  5. If installing with `tls.enabled` then there will be a `ServiceAccount` that is left behind:
-
-  ```shell-session
-  $ kubectl get serviceaccount consul-tls-init
-  NAME              SECRETS   AGE
-  consul-tls-init   1         47m
-  ```
-
-  ```shell-session
-  $ kubectl delete serviceaccount consul-tls-init
-  serviceaccount "consul-tls-init" deleted
-  ```
+You can uninstall Consul using Helm commands or the Consul K8s CLI.
```

Comment:
This looks like this is the old uninstall steps for Helm, could you copy and paste from latest? It still shows deleting `kubectl delete secret consul-controller-webhook-cert` as a step which is no longer needed.

Reply:
Yeah, I'm having some git trouble, trying to get things into the correct state

Reply:
Oh, I thought you were referring to some content regressions (I had some conflicts earlier), but you literally meant that this content needs to be updated. Where do I get this information? I looked at the Kubernetes docs and this appears to still be the correct usage: https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#delete

Reply:
Yes we should use `helm uninstall` instead of `helm delete`. 

I was saying that https://www.consul.io/docs/k8s/operations/uninstall should have the latest correct uninstall process that I merged yesterday, but yea I meant the content had old content from a previous version of the uninstall guide. We also merge 3 and 4 into one bullet as well since they are part of the same step.